Emby.Server.Implementations/SocketSharp/WebSocketSharpRequest.cs

```diff
@@ -307,11 +307,11 @@ private static string GetQueryStringContentType(IRequest httpReq)
             }
 
             format = LeftPart(format, '.');
-            if (format.Contains("json", StringComparison.OrdinalIgnoreCase))
+            if (format.ToLower().Contains("json"))
```

Comment:
Don't use `ToLower` use the overload with `StringComparison.OrdinalIgnoreCase`

Reply:
As above

Reply:
Then use `.IndexOf` instead of `.Contains`